src/main/java/io/ipfs/api/Multipart.java

```diff
@@ -24,6 +24,7 @@ public Multipart(String requestURL, String charset) {
             httpConn.setUseCaches(false);
             httpConn.setDoOutput(true);
             httpConn.setDoInput(true);
+            httpConn.setChunkedStreamingMode(1024 * 1024 * 10);
```

Comment:
10G seems to make sense. Let's just take note that we might need to do the same on the other IPFS libraries.

Reply:
10MB only :)